Switch Sourcify tests to the Chiado network

Sourcify deprecated Sokol network, so we need to update tests and use another testnet, for example Chiado
```
2023-08-07T07:01:49.7605403Z   1) test /api/v2/smart-contracts/{address_hash}/verification/via/sourcify verify contract from sourcify repo (BlockScoutWeb.API.V2.VerificationControllerTest)
2023-08-07T07:01:49.7612057Z ##[error]     test/block_scout_web/controllers/api/v2/verification_controller_test.exs:224
2023-08-07T07:01:49.7613114Z      Assertion failed, no matching message after 120000ms
2023-08-07T07:01:49.7613738Z      The following variables were pinned:
2023-08-07T07:01:49.7614128Z        topic = "addresses:0x18d89c12e9463be6343c35c9990361ba4c42afc2"
2023-08-07T07:01:49.7614497Z      Showing 4 of 4 messages in the mailbox
2023-08-07T07:01:49.7614932Z      code: assert_receive %Phoenix.Socket.Message{
2023-08-07T07:01:49.7615383Z              payload: %{status: "success"},
2023-08-07T07:01:49.7615768Z              event: "verification_result",
2023-08-07T07:01:49.7616125Z              topic: ^topic
2023-08-07T07:01:49.7616433Z            }
2023-08-07T07:01:49.7616934Z      mailbox:
2023-08-07T07:01:49.7617594Z        pattern: %Phoenix.Socket.Message{
2023-08-07T07:01:49.7617982Z                   payload: %{status: "success"},
2023-08-07T07:01:49.7618314Z                   event: "verification_result",
2023-08-07T07:01:49.7618636Z                   topic: ^topic
2023-08-07T07:01:49.7618906Z                 }
2023-08-07T07:01:49.7619200Z        value:   {#Reference<0.3192504108.1956380674.144618>,
2023-08-07T07:01:49.7619495Z                  {200,
2023-08-07T07:01:49.7619750Z                   [
2023-08-07T07:01:49.7620214Z                     {"content-type",
2023-08-07T07:01:49.7620615Z                      "application/json; charset=utf-8"},
2023-08-07T07:01:49.7620978Z                     {"cache-control",
2023-08-07T07:01:49.7621378Z                      "max-age=0, private, must-revalidate"},
2023-08-07T07:01:49.7621908Z                     {"x-request-id", "F3kHsINNtvwRnkYAAJyi"},
2023-08-07T07:01:49.7622474Z                     {"access-control-allow-credentials", "true"},
2023-08-07T07:01:49.7623011Z                     {"access-control-allow-origin", "*"},
2023-08-07T07:01:49.7623523Z                     {"access-control-expose-headers", ""}
2023-08-07T07:01:49.7623971Z                   ], "{\"message\":\"Verification started\"}"}}
2023-08-07T07:01:49.7624252Z 
2023-08-07T07:01:49.7624609Z        pattern: %Phoenix.Socket.Message{
2023-08-07T07:01:49.7625189Z                   payload: %{status: "success"},
2023-08-07T07:01:49.7625585Z                   event: "verification_result",
2023-08-07T07:01:49.7625969Z                   topic: ^topic
2023-08-07T07:01:49.7626305Z                 }
2023-08-07T07:01:49.7626802Z        value:   {:plug_conn, :sent}
2023-08-07T07:01:49.7627161Z 
2023-08-07T07:01:49.7627428Z        pattern: %Phoenix.Socket.Message{
2023-08-07T07:01:49.7627948Z                   payload: %{status: "success"},
2023-08-07T07:01:49.7628346Z                   event: "verification_result",
2023-08-07T07:01:49.7628710Z                   topic: ^topic
2023-08-07T07:01:49.7629049Z                 }
2023-08-07T07:01:49.7629413Z        value:   %Phoenix.Socket.Broadcast{
2023-08-07T07:01:49.7629870Z                   topic: "addresses:0x18d89c12e9463be6343c35c9990361ba4c42afc2",
2023-08-07T07:01:49.7630301Z                   event: "verification_result",
2023-08-07T07:01:49.7630659Z                   payload: %{
2023-08-07T07:01:49.7631003Z                     result: {:error,
2023-08-07T07:01:49.7631355Z                      #Ecto.Changeset<
2023-08-07T07:01:49.7631705Z                        action: nil,
2023-08-07T07:01:49.7632061Z                        changes: %{},
2023-08-07T07:01:49.7632405Z                        errors: [
2023-08-07T07:01:49.7636659Z                          files: {"Chain 77 not supported for verification!",
2023-08-07T07:01:49.7636997Z                           []}
2023-08-07T07:01:49.7637243Z                        ],
2023-08-07T07:01:49.7637569Z                        data: #Explorer.Chain.SmartContract<>,
2023-08-07T07:01:49.7638022Z                        valid?: false
2023-08-07T07:01:49.7638283Z                      >}
2023-08-07T07:01:49.7638643Z                   }
2023-08-07T07:01:49.7638889Z                 }
2023-08-07T07:01:49.7639032Z 
2023-08-07T07:01:49.7639187Z        pattern: %Phoenix.Socket.Message{
2023-08-07T07:01:49.7639652Z                   payload: %{status: "success"},
2023-08-07T07:01:49.7639973Z                   event: "verification_result",
2023-08-07T07:01:49.7640383Z                   topic: ^topic
2023-08-07T07:01:49.7640630Z                 }
2023-08-07T07:01:49.7641026Z        value:   %Phoenix.Socket.Message{
2023-08-07T07:01:49.7641632Z                   topic: "addresses:0x18d89c12e9463be6343c35c9990361ba4c42afc2",
2023-08-07T07:01:49.7642738Z                   event: "verification_result",
2023-08-07T07:01:49.7643157Z                   payload: %{
2023-08-07T07:01:49.7643416Z                     errors: %{
2023-08-07T07:01:49.7643918Z                       files: ["Chain 77 not supported for verification!"]
2023-08-07T07:01:49.7644300Z                     },
2023-08-07T07:01:49.7644553Z                     status: "error"
2023-08-07T07:01:49.7644820Z                   },
2023-08-07T07:01:49.7645067Z                   ref: nil,
2023-08-07T07:01:49.7645447Z                   join_ref: 40067
2023-08-07T07:01:49.7646120Z                 }
2023-08-07T07:01:49.7646610Z      stacktrace:
2023-08-07T07:01:49.7647309Z        test/block_scout_web/controllers/api/v2/verification_controller_test.exs:264: (test)
2023-08-07T07:01:49.7647650Z 
```

--- a/.github/workflows/config.yml
+++ b/.github/workflows/config.yml
@@ -625,7 +625,7 @@ jobs:
           PGUSER: postgres
           ETHEREUM_JSONRPC_CASE: "EthereumJSONRPC.Case.Nethermind.Mox"
           ETHEREUM_JSONRPC_WEB_SOCKET_CASE: "EthereumJSONRPC.WebSocket.Case.Mox"
-          CHAIN_ID: "77"
+          CHAIN_ID: "10200"
           API_RATE_LIMIT_DISABLED: "true"
           ADMIN_PANEL_ENABLED: "true"
           ACCOUNT_ENABLED: "true"

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,6 +10,7 @@
 
 ### Fixes
 
+- [#8147](https://github.com/blockscout/blockscout/pull/8147) - Switch sourcify tests from POA Sokol to Gnosis Chiado
 - [#8145](https://github.com/blockscout/blockscout/pull/8145) - Handle negative holders count in API v2
 - [#8040](https://github.com/blockscout/blockscout/pull/8040) - Resolve issue with Docker image for Mac M1/M2
 - [#8060](https://github.com/blockscout/blockscout/pull/8060) - Fix eth_getLogs API endpoint

--- a/apps/block_scout_web/test/block_scout_web/controllers/api/rpc/contract_controller_test.exs
+++ b/apps/block_scout_web/test/block_scout_web/controllers/api/rpc/contract_controller_test.exs
@@ -712,7 +712,7 @@ defmodule BlockScoutWeb.API.RPC.ContractControllerTest do
       params = %{
         "module" => "contract",
         "action" => "verify_via_sourcify",
-        "addressHash" => "0x18d89C12e9463Be6343c35C9990361bA4C42AfC2"
+        "addressHash" => "0xf26594F585De4EB0Ae9De865d9053FEe02ac6eF1"
       }
 
       response =
@@ -732,14 +732,14 @@ defmodule BlockScoutWeb.API.RPC.ContractControllerTest do
       _created_contract_address =
         insert(
           :address,
-          hash: "0x18d89C12e9463Be6343c35C9990361bA4C42AfC2",
+          hash: "0xf26594F585De4EB0Ae9De865d9053FEe02ac6eF1",
           contract_code: smart_contract_bytecode
         )
 
       params = %{
         "module" => "contract",
         "action" => "verify_via_sourcify",
-        "addressHash" => "0x18d89C12e9463Be6343c35C9990361bA4C42AfC2"
+        "addressHash" => "0xf26594F585De4EB0Ae9De865d9053FEe02ac6eF1"
       }
 
       get_implementation()

--- a/apps/block_scout_web/test/block_scout_web/controllers/api/v2/verification_controller_test.exs
+++ b/apps/block_scout_web/test/block_scout_web/controllers/api/v2/verification_controller_test.exs
@@ -222,7 +222,7 @@ defmodule BlockScoutWeb.API.V2.VerificationControllerTest do
     end
 
     test "verify contract from sourcify repo", %{conn: conn} do
-      address = "0x18d89C12e9463Be6343c35C9990361bA4C42AfC2"
+      address = "0xf26594F585De4EB0Ae9De865d9053FEe02ac6eF1"
 
       _contract = insert(:address, hash: address, contract_code: "0x01")
 

